packages/amplify_auth_cognito/lib/src/CognitoSignIn/CognitoSignInWithWebUIOptions.dart

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import 'package:amplify_auth_plugin_interface/amplify_auth_plugin_interface.dart';
+
+class CognitoSignInWithWebUIOptions extends SignInWithWebUIOptions {
+  bool? isPreferPrivateSession;
+  CognitoSignInWithWebUIOptions({this.isPreferPrivateSession}) : super();
```

Comment:
Prefer default value here

```suggestion
  const CognitoSignInWithWebUIOptions({this.isPreferPrivateSession = false});
```

Reply:
Thank you! I have updated the code